app/views/green_lanes/category_assessments/_categories.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @goods_nomenclature.filter_by_category('1').any?%>
+  <h2>Category 1 Assessments</h2>
+  <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+    <% @goods_nomenclature.filter_by_category('1').each do |category| %>
+      <%= render 'green_lanes/category_assessments/category', category: category %>
```

Comment:
Minor but for future reference theres a `collection:` option when rendering partials, so you can do

```ruby
render partial: 'green_lanes/category_assessments/category', 
       collection: @goods_nomenclature.filter_by_category('1')
```

Reply:
Done